libfreerdp/locale/timezone.c

```diff
@@ -1523,7 +1523,9 @@ char* freerdp_get_unix_timezone_identifier()
 		return tzid;
 	}
 
-#if defined(__FreeBSD__) || defined(__OpenBSD__)
+#if defined(ANDROID)
+	fp = popen("getprop persist.sys.timezone", "r");
```

Comment:
as @mfleisz said pclose should later be used if popen was used for opening the i/o stream. Otherwise the behavior is undefined.